reana_workflow_engine_yadage/cli.py

```diff
@@ -53,6 +54,11 @@ def load_yadage_operational_options(ctx, param, operational_options):
     operational_options["initdir"] = os.path.join(
         workflow_workspace, operational_options.get("initdir", "")
     )
+
+    operational_options["initfiles"] = operational_options.get("initfiles", [])
+    for i, initfile in enumerate(operational_options["initfiles"]):
```

Comment:
Normally list comprehensions look more complicated than they should, but I feel in this case it would be more illustrative than the explicit loop:
```python
operational_options["initfiles"] = [
    os.path.join(workflow_workspace, initfile) for initfile in operational_options.get("initfiles", [])]
```

There is no need to change anything if you don't think the same after reading the comment :)

Reply:
I actually prefer list comprehensions, not sure why I didn't think of it here :) I'll change it.